blog_post_collaboration/kedro_ibis.md

```diff
@@ -0,0 +1,263 @@
+# From production-ready to production: building scalable data pipelines with Kedro and Ibis
+
+In your Kedro journey, have you ever...
+
+* ...slurped up large amounts of data into memory, instead of pushing execution down to the source database/engine?
+* ...implemented a proof-of-concept solution in 3-4 months on data extracts, and then struggled massively when you needed to move to running against the production databases and scale out?
```

Comment:
kind of related, not sure how to phrase, but similarly "prototypes on pandas and had to convert code over to PySpark/Snowpark/other database-native Python dataframes?"

Reply:
reading on this is already addressed; perhaps a bullet point up here? but 🤷